Automattic/wp-calypso
# Update/link in bio flow

## Proposed Changes

Organize the steps order for the whole Link in bio flow. Some steps are fake until they get complete.

## Testing Instructions

- Pull this branch and `yarn start` from the root folder
- Start at the beginning of the flow http://calypso.localhost:3000/setup/intro?flow=link-in-bio
- The flow order should be: ( according to the latest design )

1. Intro / Welcome 
2. If logged out - > account creation;   If not Choose a design (fake step for now)
3. Link in bio setup
4. Choose a domain 
5. Choose a plan
6. Pre checkout;  `fake processing step for now`
7. Checkout 
8. Pre launchpad `fake for now`
9. Launchpad  

  - test with logged in and logged out
  - test purchase and free plan


<img width="723" alt="image" src="https://user-images.githubusercontent.com/2653810/186198381-3cb65e65-89f8-4c32-9f93-664e5214f88f.png">

Related to #66576
